src/app/ui/landing-page/landing-page.js

```diff
@@ -196,6 +201,7 @@ export class LandingPage extends ViewPlugin {
             <div class="${css.enviroments} pt-2">
               <button class="btn btn-lg btn-secondary mr-3" onclick=${() => startSolidity()}>Solidity</button>
               <button class="btn btn-lg btn-secondary mr-3" onclick=${() => startVyper()}>Vyper</button>
+              <button class="btn btn-lg btn-secondary mr-3" onclick=${() => startLexon()}>Lexon</button>
```

Comment:
sry we can't really do that in this PR, changing the UI this way will require some thinking from our side. can that be removed for the moment?

Reply:
sry, removed.